### ISSUE TITLE
RUN @LSF cluster: MARVEL

### ISSUE THREAD

**Issue by hoelzer** [replikation/What_the_Phage#17]
````
[e4/19acd4] process > marvel_wf:input_suffix_check (1) [100%] 1 of 1 ✔
[8e/c04eb1] process > marvel_wf:marvel (1)             [100%] 1 of 1, failed: 1 ✘
[-        ] process > marvel_wf:filter_marvel          -
Error executing process > 'marvel_wf:marvel (1)'

Caused by:
  Process `marvel_wf:marvel (1)` terminated with an error exit status (127)

Command executed:

  mkdir fasta_dir_T7_draft
        cp T7_draft.1.fa T7_draft.2.fa T7_draft.3.fa T7_draft.4.fa T7_draft.5.fa T7_draft.6.fa fasta_dir_T7_draft/
        # Marvel
        marvel_bins.py -i fasta_dir_T7_draft -t 8 > results.txt
        # getting contig names
        filenames=$(grep  "T7_draft\." results.txt | cut -f2 -d " ")
        while IFS= read -r samplename ; do
         head -1 fasta_dir_T7_draft/${samplename}.fa >> T7_draft.txt
        done < <(printf '%s
  ' "${filenames}")

Command exit status:
  127

Command output:
  (empty)

Command error:
  .command.sh: line 5: marvel_bins.py: command not found
````

**Comment by hoelzer** [comment]
````
singularity exec docker://multifractal/marvel:0.1 /bin/bash -c "cd $PWD; /bin/bash -ue /hps/nobackup2/production/metagenomics/mhoelzer/nextflow-work-mhoelzer/8e/c04eb167016e1119f0ac24d7ee5093/.command.sh"
````

again works. So not using the *.img singularity file generated during nextflow execution (like for PPRMeta #16) seems to work.

**Comment by replikation** [comment]
i would suggest now to add this to nextflow as an issue, or you could ask here in gitter:
https://gitter.im/nextflow-io/nextflow#

**Comment by hoelzer** [comment]
This is solved now like described in #16
